src/equations/2d/ideal_glm_mhd.jl

```diff
@@ -453,14 +453,25 @@ function flux_hll(u_ll, u_rr, orientation, equation::IdealGlmMhdEquations2D)
   return SVector(f1, f2, f3, f4, f5, f6, f7, f8, f9)
 end
 
-
-# strong form of nonconservative flux on a side, e.g., the Powell term
-#     phi^L 1/2 (B^L+B^R) normal - phi^L B^L normal = phi^L 1/2 (B^R-B^L) normal
-# OBS! 1) "weak" formulation of split DG already includes the contribution -1/2(phi^L B^L normal)
-#         so this routine only adds 1/2(phi^L B^R normal)
-#         analogously for the Galilean nonconservative term
-#      2) this is non-unique along an interface! normal direction is super important
-function noncons_interface_flux(u_left, u_right, orientation, equation::IdealGlmMhdEquations2D)
+"""
+    noncons_interface_flux(u_left, u_right, orientation, mode, equation::IdealGlmMhdEquations2D)
```

Comment:
In the DG solver we always spell out `nonconservative`, so to be Mr. Consistency here, maybe it would make sense to rename it here as well (even though it was named differently before)?
```suggestion
    nonconservative_interface_flux(u_left, u_right, orientation, mode, equation::IdealGlmMhdEquations2D)
```

Reply:
We will leave this naming convention of `noncons_interface_flux` for the time being